src/lib/logger.js

```diff
@@ -25,12 +25,11 @@ const METHODS = ["debug", "log", "warn", "error"];
 let isCLI = false;
 let level = LEVELS.log;
 
+
 // create a prefixing & colorizing wrapper around console for use in non-CLIs
 const wrappedConsole = Object.create(console);
 METHODS.forEach(method => {
-    const color = CONSOLE_COLORS[method]
-        ? CONSOLE_COLORS[method]
-        : v => v;
+    const color = CONSOLE_COLORS[method];
```

Comment:
Can you motivate this change? I agree it _seems_ sensible but having the fallback doesn't hurt I think. If you'd want to avoid the fallback I'd refactor this file such that, e.g., the `METHODS` are derived from the `CONSOLE_COLORS` but I think that's beyond the scope of this Pull Request as it would slow down the merge process of the disable colors functionality (feel free to open a separate Pull Request for that though).